packages/fast-components-react-base/src/image/image.props.ts

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import { IImageClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts";
+
+export interface IImageHandledProps {
+    /**
+     * The HTML alt attribute value is important for overall accessibility, providing a textual
+     * alternative to non-text content
+     */
+    alt: string;
+
+    /**
+     * The HTML itemScope attribute defines the scope of associated metadata and is used for associating the
+     * element context with definitions on schema.org (important for seach engine optimization)
+     */
+    itemScope?: boolean;
+
+    /**
+     * The round image option
+     */
+    round?: boolean;
+
+    /**
+     * The HTML src attribute for the image element (specifying src will chose the image element rather than the picture element)
+     */
+    src?: string;
+
+    /**
```

Comment:
we may need to be more specific in our comments here since everything is optional. Since srcSet is possible on picture as well we should specify that these are for the simple instance. In picture a user would likely pass this as part of the srcset for the source element (VP1, VP2, etc) - @Wilto to confirm that works.

Reply:
(Sorry, late on this reply!) 

Yeah, that tracks—`srcset` on `<source>` can be used exactly like `src` on `img`, if you don’t pass any additional `srcset` descriptors:

```html
<picture>
<source srcset="the/source.jpg">
<img src="…" alt="…">
</picture>
```

is functionally equivalent to:

```html
<img src="the/source.jpg" alt="…">
```